client/src/pages/Results.js

```diff
@@ -0,0 +1,117 @@
+import React, { useState, useEffect } from "react";
+import { BrowserRouter } from "react-router-dom";
+import GlobalStyles from "../GlobalStyles";
+import styled from "@emotion/styled";
+import SearchHeader from "../components/SearchHeader";
+import Footer from "../components/Footer";
+import List from "../components/List";
+import { fetchResults } from "../api/results";
+import LocationSrc from "../assets/images/secondBite_Icon_Location.svg";
+import DateSrc from "../assets/images/secondBite_Icon_Date.svg";
+import TimeSrc from "../assets/images/secondBite_Icon_Time.svg";
+import ArrowSrc from "../assets/images/secondBite_Icon_ArrowRight.svg";
+import fruitsSrc from "../assets/images/sb_icon_fruits.svg";
+
+//import SearchInput from "../components/SearchInput";
+
+const Main = styled.div`
+  margin: 4em 0;
+  display: grid;
+`;
+
+const ListItem = styled.a`
+  display: grid;
+  grid-template: 4em 1fr 1em / 3em 1fr 1em;
+  align-items: center;
+  margin: 1em;
+  border-bottom: 1px solid #eeeeee;
+  text-decoration: none;
+  color: var(--font-semi-dark);
+`;
+
+const CategoryImage = styled.img`
+  grid-column: 1 / 2;
+  max-height: 2em;
+`;
+
+const Description = styled.div`
+  grid-column: 2 / 3;
+  padding-left: 1em;
+
+  img {
+    height: 1.2em;
+    padding-right: 0.3em;
+    margin-bottom: -0.2em;
+  }
+  h3,
+  p {
+    padding-bottom: 0.5em;
+  }
+  p {
+    font-size: 0.9em;
+  }
+`;
+
+const ArrowIcon = styled.img`
+  grid-column: 3 / 4;
+  max-height: 1em;
+`;
+
+export function Results() {
+  const [results, setResults] = useState([]);
+  const [isLoading, setIsLoading] = useState(true);
+  //const [query, setQuery] = useState("");
+
+  useEffect(() => {
+    async function fetchData() {
+      setIsLoading(true);
+      const offers = await fetchResults();
+      setResults(offers);
+      setIsLoading(false);
+    }
+    fetchData();
+  }, []);
+
+  if (isLoading || results === null) {
+    //return <LoadingScreen />;
+    console.log("loading screen");
+  }
+
+  /*
+  const filteredResults = results.filter((result) => {
+    return result.title.toLowerCase().match(query.toLowerCase());
+  });*/
+
+  return (
+    <>
+      <GlobalStyles />
+      <BrowserRouter>
```

Comment:
`GlobalStyles` and `BrowserRouter` should be part of your App, not of your page. Otherwise, you can not reuse these providers.

Reply:
Will fix this in later steps.